### PR TITLE
GK updates

### PR DESCRIPTION
**Pull request by SethDavenport**

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ machine:
     version: 2.7.9
 
   node:
-    version: 4
+    version: 6
```
Comment (JaKXz) [inline circle.yml]: Just curious, what was the motivation behind this change? :)

File: package.json [hunk 2]
```diff
@@ -39,18 +38,7 @@
...
-    "basscss-border": "^4.0.2",
-    "basscss-flexbox": "^1.0.1",
-    "basscss-grid": "^2.0.0",
-    "basscss-hide": "^1.0.1",
-    "basscss-layout": "^3.1.0",
-    "basscss-margin": "^1.0.7",
-    "basscss-padding": "^1.1.3",
-    "basscss-position": "^2.0.3",
-    "basscss-type-scale": "^1.0.5",
-    "basscss-typography": "^3.0.3",
```
Comment (JaKXz) [inline package.json]: Same question as above, is this taken care of magically somewhere else? Or just moving away from basscss?

File: circle.yml (hunk 1 above)
Comment (SethDavenport) [inline circle.yml]: Angular 2 peer deps + npm 2 === pain

File: package.json (hunk 2 above)
Comment (SethDavenport) [inline package.json]: npm 3 handles it magically

**Pull request merged**